R/vector_control_parameters.R

```diff
@@ -133,3 +133,59 @@ set_spraying <- function(
   parameters$spraying_ms_gamma <- ms_gamma
   parameters
 }
+
+#' @title Parameterise a habitat management strategy
+#'
+#' @description The model will simulate larval source habitat management at 
+#' `timesteps` to the entire human population. 
+#' The impact will reduce adult mosquitoes recruited from pupae to the susceptible
+#' cohort.
+#'
+#' The rapidity of the impact is captured by the lsm_factor function
+#' by changing the parameters lsm_rate and deprec_param
+#'
+#' The amount of impact in terms of reduced adult mosquito densities
+#' that the habitat management has is determined by larvi_min which is
+#' set to 1 as default when there is no impact.
+#'
+#' Habitat management effects will be randomly removed each timestep with a rate of `1 -
+#' exp(-1/habitat_management_waning)`
+#'
+#' The structure for the habitat model is documented in the
+#' working draft Sherrard-Smith et al (larviciding in Kenya)
+#'
+#' @param parameters a list of parameters to modify
+#' @param timesteps the timesteps at which to distribute lsm rounds
+#' @param larvi_min the proportion of the mosquito population reduced (0 to 1), 1 is no impact
+#' @param lsm_rate the duration til the impact is acheived
+#' @param deprec_param the speed on impact taking mosquito densities down to new-normal 
```

Comment:
I don't understand this description - what is the speed measured by and what is new-normal?

Reply:
the function reduces the proportion of mosquitoes moving from the pupae to susceptible adults cohort. The new normal (now new equilibrium) is the level of reduction set from 1 (where there is no change) to anything smaller than this down to 0 (when there is a full reduction in mosquitoes)